apps/sms/services/js/bridge_service_mixin.js

```diff
@@ -55,6 +55,10 @@
         });
       }
 
+      if (this.onConnected) {
```

Comment:
I found it as the simplest way, other alternative was smth like ```BridgeServiceMixin.ON_CONNECT = Symbol('ON_CONNECT')``` and
```
var service = {
  ...
  [BridgeServiceMixin.ON_CONNECT]() {}
  ...
}
```

What do you think?